.env.example

```diff
@@ -37,3 +37,7 @@ SCOUT_DRIVER=meilisearch
 SCOUT_QUEUE=true
 MEILISEARCH_HOST=http://meilisearch:7700
 MEILISEARCH_NO_ANALYTICS=false
+
+SENTRY_LARAVEL_DSN=
+SENTRY_TRACES_SAMPLE_RATE=1.0
```

Comment:
For the example env file you'll want to keep these a `0` so we don't miss the config and blow past the free limit.